spinnaker/src/main/java/org/bytedeco/spinnaker/presets/Spinnaker_C.java

```diff
@@ -95,6 +95,9 @@ public void map(InfoMap infoMap) {
                // Skip to avoid linker errors, somehow Spinnaker SDK does not export those functions,
                // To avoid errors like: jniSpinnaker_C.obj : error LNK2001: unresolved external symbol spinCameraForceIP
                .put(new Info("spinCameraForceIP", "spinRegisterSetEx", "spinSystemSendActionCommand", "spinInterfaceSendActionCommand").skip())
+               .put(new Info("SPINNAKER_TLPAYLOAD_TYPE_LOSSLESS_COMPRESSED").skip())
+               .put(new Info("SPINNAKER_TLPAYLOAD_TYPE_LOSSY_COMPRESSED").skip())
+               .put(new Info("SPINNAKER_TLPAYLOAD_TYPE_JPEG_LOSSLESS_COMPRESSED").skip())
```

Comment:
Let's put all that in the same entry, something like this:
```java
               .put(new Info("spinCameraForceIP", "spinRegisterSetEx", "spinSystemSendActionCommand", "spinInterfaceSendActionCommand",
                             "SPINNAKER_TLPAYLOAD_TYPE_LOSSLESS_COMPRESSED",
                             "SPINNAKER_TLPAYLOAD_TYPE_LOSSY_COMPRESSED",
                             "SPINNAKER_TLPAYLOAD_TYPE_JPEG_LOSSLESS_COMPRESSED").skip())
```

Reply:
Whoops, I've fixed that.